internal/i18n/locales/en/default.po

```diff
@@ -102,6 +102,12 @@ msgstr "SMS text message (recommended)"
 msgid "codes.issue.sms-text-message-label"
 msgstr "Patient phone number"
 
+msgid "codes.issue.sms-template-label"
```

Comment:
I _think_ you need to translate all the things now or the tests will fail.

Reply:
Added Google translate translations.